riposte-core/src/test/java/com/nike/riposte/server/handler/RoutingHandlerTest.java

```diff
@@ -182,4 +196,99 @@ public void findSingleEndpointForExecution_throws_MultipleMatchingEndpointsExcep
         // when
         handlerSpy.findSingleEndpointForExecution(requestInfoMock);
     }
+
+    @Test
+    public void doChannelRead_HttpRequest_throws_exception_when_content_length_header_greater_than_configured_global_request_limit() {
+        // given
+        doReturn(null).when(endpointMock).maxRequestSizeInBytesOverride();
+
+        maxRequestSizeInBytes = 10;
```

Comment:
Do you need to specify `maxRequestSizeInBytes` and create your own `handlerSpy`? Isn't this how `handlerSpy` is already setup?